frontend/src/pictureBookView/pictureBookView.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { Navbar, Footer, LoadingPage } from '../UILibrary/components';
+
+const capitalize = (word) => {
+    return word.charAt(0).toUpperCase() + word.slice(1);
+};
+
+export class PictureBookView extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            pictureData: null,
+            showModal: false,
+            pictureBookStory: '',
+            pictureBookWords: null,
+            submitted: false,
+        };
+        this.modalHandler = this.modalHandler.bind(this);
+    }
+
+    componentDidMount = async () => {
+        try {
+            const apiURL = `/api/get_picturebook_prompt/${this.props.textID}/`
+                + `${this.props.partOfSpeech}`;
+            const response = await fetch(apiURL);
+            const pictureData = await response.json();
+            this.setState({ pictureData });
+            document.addEventListener('keydown', this.handleKeyDown, true);
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    createPictureBook = async (input) => {
```

Comment:
One comment as a thing to think about after we merge this in. What should this function be doing? As it stands, it simply makes a GET request with `undefined` as the input (as we don't pass anything when we call this in `handleSubmit`). Presumably there should be some more substantive action there. Since the rest of this is good to go, let's call this something to return to later.